docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: docs
+title: Getting Started
+permalink: /documentation/getting-started/
+---
+# Getting Started
+
+This guide provides step-by-step instructions on how to create a functional web test with TestCafe and contains the following sections.
+
+* [Installing TestCafe](#installing-testcafe)
+* [Writing Test](#writing-test)
+  * [General Code Structure](#general-code-structure)
+  * [Performing Actions on Page](#performing-actions-on-page)
+  * [Observing Page State](#observing-page-state)
+  * [Assertions](#assertions)
+* [Running Test](#running-test)
+* [Viewing Test Results](#viewing-test-results)
+
+## Installing TestCafe
+
+You can install TestCafe into a website project you are going to test.
+
+First, make sure that [Node.js](https://nodejs.org/) and [npm](https://www.npmjs.com/) are installed on your computer. Then go to the tested project directory and run a single command -
+`npm install --save-dev testcafe`.
+
+This tutorial will demonstrate how to test not a website project, but the published web page - [http://testcafe.devexpress.com/Example](http://testcafe.devexpress.com/Example).
+To create tests for this page, create the *my-project* directory anywhere on your computer and install TestCafe into it. You can do this from a command line.
+
+```bash
+mkdir my-project
+cd my-project
+npm install --save-dev testcafe
+```
+
+## Writing Test
+
+You can organize your tests into any directory on your computer. It is a good practice to create the *test* directory as a subdirectory of the website project you are going to test and place tests there.
+
+```bash
+cd my-project
+mkdir test
+```
+
+To create a test, go to the *test* directory, create a new .js file and add your code to it.
+
+### General Code Structure
+
+TestCafe tests must be organized into fixtures within a .js file.
+You declare fixtures by using the `fixture` function.
+You can optionally specify a start page URL for a fixture by using the [page](../test-api/test-code-structure.md#specifying-the-start-webpage) function.
+In this tutorial, you will create a test for the [http://testcafe.devexpress.com/example](http://testcafe.devexpress.com/example) sample page.
+
+Test code is placed within the `test` function.
+
+```js
+fixture `Getting Started`;
+    .page('http://testcafe.devexpress.com/example');
+
+test('MyTest', async t => {
+    //Test code
+});
+```
+
+### Performing Actions on Page
+
+Every test should be capable of interacting with a page content. To perform user actions, TestCafe provides a number of [actions](../test-api/actions.md): `click`, `hover`, `typeText`, `upload`, etc.
+They can be called in a chain.
+
+The following fixture contains a simple test that types a developer name into a text editor and then clicks the Submit button.
+You can try running this test against the sample page by yourself.
```

Comment:
I have no idea how to run tests yet. Maybe it makes sense to move `running test` and `Viewing test results` sections after we have the first runable code snippet?